src/test/java/examples/ExamplesTest.java

```diff
@@ -0,0 +1,11 @@
+package examples;
```

Comment:
generally, the package is the same as the groupID-artifact id for consistency, but not required